crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -0,0 +1,677 @@
+//! Built-in verbs for the Bevy Remote Protocol.
+
+use std::any::TypeId;
+
+use anyhow::{anyhow, Result as AnyhowResult};
+use bevy_ecs::{
+    component::ComponentId,
+    entity::Entity,
+    query::QueryBuilder,
+    reflect::{AppTypeRegistry, ReflectComponent},
+    system::In,
+    world::{EntityRef, EntityWorldMut, FilteredEntityRef, World},
+};
+use bevy_hierarchy::BuildChildren as _;
+use bevy_reflect::{
+    serde::{ReflectSerializer, TypedReflectDeserializer},
+    PartialReflect, TypeRegistration, TypeRegistry,
+};
+use bevy_utils::HashMap;
+use serde::de::DeserializeSeed as _;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::{error_codes, BrpError, BrpResult};
+
+/// `bevy/get`: Retrieves one or more components from the entity with the given
+/// ID.
+///
+/// The server responds with a [`BrpGetResponse`].
+#[derive(Serialize, Deserialize, Clone)]
+pub struct BrpGetParams {
```

Comment:
I think that these types would be clearer as `BrpGetRequest` and so on, to mirror `BrpGetResponse`.

Reply:
Yeah, I feel a bit divided on this; they are named like this because the `BrpRequest` type encompasses the entire request, and the params are what is method-specific (and corresponds to the contents of `params` in the request).